### PR TITLE
Create a welsh translation of participation interface

### PR DESCRIPTION
**Pull request by patcon**
Submitted by Gwilym Morus-Baird, attendee of open call who is not accustomed to GitHub.

(included commit cherry-picked from https://github.com/g0v-network/polis/pull/2 to get e2e tests passing again)

Will submit change upstream after.

## Demo Preview

http://104.236.45.174.xip.io/5z4cdaxiah?ui_lang=cy

Screen Shot 2021-04-09 at 12 41 19 AM

Got this working here on a temporary server that I'll host until this pending pull request is closed. It will then be deleted.

## Reference

Steps for fully adding translations: https://github.com/compdemocracy/polis/discussions/761#discussioncomment-279671

Call notes:  https://hackmd.io/@patcon/conversa-calls/https%3A%2F%2Fhackmd.io%2F%40patcon%2FryhIoUufd

### CODE REVIEW

File: client-participation/js/strings/cy.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2012-present, The Authors. This program is free software: you can redistribute it and/or  modify it under the terms of the GNU Affero General Public License, version 3, as published by the Free Software Foundation. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+var s = {};
+
+s.agree = "Cytuno";
+s.disagree = "Anghytuno";
+s.pass = "Dim diddordeb / Ansicr";
+
+s.importantCheckbox = "Mae’r sylw yma’n bwysig";
```
Comment (patcon) [inline client-participation/js/strings/cy.js]: should prob remove all these fancy quotes, and markdown processor use smartquotes to do this on-the-fly

**Comment by patcon** [comment]
@daniel-skywalker hey! while it isn't a polis instance that you should actually use for outside conversations, if you're just looking to see how the translations look like within the interface, then pls do feel welcome to explore: http://104.236.45.174.xip.io/5z4cdaxiah?ui_lang=he

This Polis instance will likely go away after this is merged (also, it's definitely _not_ secure or something you should use for a conversation)

EDIT: You should also consider joining the Discord chat server for the unofficial Polis User Group, where I think you'll find more folks are interested in having conversations about deployment, compared to the official gitter chat/issues. Details here: https://bit.ly/conversa-calls

**Comment by daniel-skywalker** [comment]
Thanks Patrick!
I'm actually just interested in the actual Hebrew translation. There are
several upcoming events that could benefit from this amazing tool. For now
I'm not counting on the tool but am staying closely tuned!
Daniel

**Review by ballPointPenguin — APPROVED** [review]
This looks right

**Comment by patcon** [comment]
To clarify, just waiting to hear from Gwilym that this is good, and no more text revisions, and then we can close this PR (since no need to merge here) and submit a PR upstream where it belongs :)

**Comment by patcon** [comment]
Closing now that https://github.com/compdemocracy/polis/pull/971 is merged 🎉

**Pull request closed without merging**